Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "netscanner"
-version = "0.6.3"
+version = "0.6.4"
```

Comment:
Please don't change the App version - I will happen when new feature, bugfix or someone PR will be included in new release.
Thanks.